.pre-commit-config.yaml

```diff
@@ -72,47 +59,28 @@ repos:
   hooks:
   - id: remove-tabs
 
+# Avoid directional quotes
 - repo: https://github.com/sirosen/texthooks
   rev: "0.5.0"
   hooks:
   - id: fix-ligatures
   - id: fix-smartquotes
 
-# Autoremoves unused imports
-- repo: https://github.com/hadialqattan/pycln
-  rev: "v2.1.3"
+# Ruff, the Python auto-correcting linter written in Rust
+- repo: https://github.com/charliermarsh/ruff-pre-commit
+  rev: v0.0.247
```

Comment:
This version is already out of date please autoupdate

Reply:
You also need another arg now to provide a non-zero exit code when fixits are applied.

Reply:
I don't think pre-commit cares - it can detect changed files.

Reply:
If not, this flag needs to be added to the pre-commit hook under `entry`.